PAbishai/Ride-My-Way
# #158423201 API to create a ride offer

**What does this PR do**
Make API to fetch all ride offers

**Description of tasks to be completed?**
POST /api/v1/rides should get all ride offers

**How should this be manually tested?**
Clone the repo
Create a virtualenv - **_#virtualenv Ride-My-Way_**
cd into the repo and activate it
run **_pip install -r requirements.txt_**
cd into the resources folder
run **_pytests_**
the test should pass
comment out some functions in apis.py
run pytest again
the test should fail
run **_python apis.py_**
follow localhost:5000/api/v1/rides or the port you have for flask
it should give a json list of all the rides with the code 200

**What are the relevant pivotal tracker stories**
[#158423201](https://www.pivotaltracker.com/story/show/158422577)
